launch/launch/actions/log_info.py

```diff
@@ -1,63 +0,0 @@
-# Copyright 2018 Open Source Robotics Foundation, Inc.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-"""Module for the LogInfo action."""
-
-from typing import List
-
-import launch.logging
-
-from ..action import Action
-from ..frontend import Entity
-from ..frontend import expose_action
-from ..frontend import Parser  # noqa: F401
-from ..launch_context import LaunchContext
-from ..some_substitutions_type import SomeSubstitutionsType
-from ..substitution import Substitution
-from ..utilities import normalize_to_list_of_substitutions
-
-
-@expose_action('log')
-class LogInfo(Action):
```

Comment:
The downside to removing this is that, if someone is importing `LogInfo` by doing:

```py
from launch.actions.log_info import LogInfo
```

instead of:

```py
from launch.actions import LogInfo
```

it will just break. It's probably just a small percentage of imports, but I think it would still be worth keeping this file and re-exporting `LogInfo`. You could also add a warning in this file so that it gets printed if the file is imported directly (I _think_ that should work). It's probably overkill, but it's not hard to do, so might as well.